metadoc-js/webpack.config.js

```diff
@@ -60,18 +55,24 @@ module.exports = merge(config, {
     ]
   },
   plugins: [
-    ExtractCss,
     new CopyWebpackPlugin([
       {
         from: path.resolve(MonacoEditorBaseDir, 'vs'),
         to: 'vs',
         ignore: [ 'basic-languages/**/*', 'language/**/*' ]
       }
     ]),
+    new MiniCssExtractPlugin({
+      filename: DevMode ? '[name].css' : '[name].[hash].css',
+      chunkFilename: DevMode ? '[id].css' : '[id].[hash].css',
+    }),
     new HtmlWebpackPlugin({
       inject: true,
       favicon: ResourcesDir + '/images/favicon.png',
       template: ResourcesDir + '/index.html',
+      // Load the index bundle which contains Monaco before the Scala.js bundle.
+      chunks: ['index', 'metadoc-js-opt'],
+      chunksSortMode: 'manual',
```

Comment:
I'm not sure this is strictly needed, but it was necessary when I played around with using the monaco-editor-webpack-plugin.